src/vivarium_public_health/population/add_new_birth_cohorts.py

```diff
@@ -56,50 +49,52 @@ def add_new_birth_cohort(self, event):
 
 
 class FertilityCrudeBirthRate:
-    """Population-level model of births using Crude Birth Rate.
-    Attributes
-    ----------
-    randomness : `randomness.RandomStream`
-        A named stream of random numbers bound to vivarium's common
-        random number framework.
+    """Population-level model of births using crude birth rate.
+
     Notes
     -----
-    The OECD definition of Crude Birthrate can be found on their
+    The OECD definition of crude birth rate can be found on their
     website_, while a more thorough discussion of fertility and
     birth rate models can be found on Wikipedia_ or in demography
     textbooks.
     .. _website: https://stats.oecd.org/glossary/detail.asp?ID=490
     .. _Wikipedia: https://en.wikipedia.org/wiki/Birth_rate
     """
+
+    configuration_defaults = {
+        'fertility': {
+            'time_dependent_live_births': True,
+            'time_dependent_population_fraction': False,
+        }
```

Comment:
It definitely needs some comments. I don't really get what these flags mean unless I dig down the codes.